gstreamer/gstmedia.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+from gi.repository import Gst as gst
+from gi.repository import GLib
+import gi
+gi.require_version('Gst', '1.0')
+gi.require_version('GLib', '2.0')
+
+
+class GstMedia():
+    """
+    Class that creates the GStreamer handler
+    """
+
+    def __init__(self):
+        """
```

Comment:
still not accurate. This sentence is okay for the constructor, but the rest of the docstring is for the class.